frontend/src/metabase/components/Select.jsx

```diff
@@ -4,11 +4,22 @@ import ColumnarSelector from "metabase/components/ColumnarSelector.jsx";
 import Icon from "metabase/components/Icon.jsx";
 import PopoverWithTrigger from "metabase/components/PopoverWithTrigger.jsx";
 
+import cx from "classnames";
+
 export default class Select extends Component {
+    constructor(props, context) {
+        super(props, context);
+        this.state = {
+            value: null
+        };
+    }
+
     static propTypes = {
         value: PropTypes.any,
+        changeOnSelect: PropTypes.bool,
         options: PropTypes.array.isRequired,
         placeholder: PropTypes.string,
+        triggerClasses: PropTypes.string,
```

Comment:
The issue with using className here is it's already being used and could change the behavior of existing instances of Select that depend on the hardcoded AdminSelect class being applied. I could go one step further and refactor AdminSelect into the classNames of each individual usage of Select though.